IncludeToolbox/IncludeToolbox.csproj

```diff
@@ -8,6 +8,9 @@
     </NuGetPackageImportStamp>
     <UseCodebase>true</UseCodebase>
     <TargetFrameworkProfile />
+    <StartAction>Program</StartAction>
+    <StartProgram>$(DevEnvDir)\devenv.exe</StartProgram>
+    <StartArguments>/rootsuffix Exp</StartArguments>
```

Comment:
By default, it puts those in the proj.user file, which is ignored. The .user file still overrides the proj, but you kinda have to force it manually into the actual project to get it to take.

It's one thing I really don't like about the project system, because as a user it isn't clear if you're actually making changes or just making "preference" changes for yourself alone.